src/Box.hs

```diff
@@ -51,6 +49,7 @@ len l =
         Text s -> length s
         Row ls -> sum $ map len ls
         Space -> 1
+        -- Tab -> ??? 0 ? TODO
```

Comment:
Just realized what to do about this:  `len :: Line -> Int` should be replaced with `lineLength :: Int -> Line -> Int` where the first int is the start column, then it makes sense for all cases including `Tab`